### DISCUSSION TITLE
Translated Names 테스트

### DISCUSSION THREAD

**Discussion by RimWorldKorea** [RimWorldKorea/RMK#687] [일반]
[외지인/정착민 이름 번역모드 240416](https://gall.dcinside.com/m/rimworld/488011)

이거 RMK에 포함시키려고 하는데 시간 있는 사람 [bus 브랜치](https://github.com/RimWorldKorea/RMK/tree/bus) 받아서 테스트 좀
하모니 넣으면 같이 적용됨

난 문제 없이 되는 것 같던데 혹시 몰라서

@zilrty @Nubula01

**Comment by zilrty** [comment]
ㅇㅇ 매우 적용 잘됨
